packages/utilities/src/utilities/sameheight/sameheight.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Utility that sets an array of elements to the same height.
+ *
+ * @example
+ * import {sameheight} from '@carbon/ibmdotcom-utilities';
+ *
+ * sameheight(ElementArray);
+ *
+ * @param {Array} elemCollection Html objects array
+ */
+function sameheight(elemCollection) {
+  /**
+   * Internal function made for avoiding adding more eventlisteners
+   *
+   * @param {Array} elemArr array of elements to be alligned
+   */
+  function setHeight(elemArr) {
```

Comment:
I typically don't like to see nested functions. Can you create this as perhaps a private function outside of this?